lipo_app.py

```diff
@@ -0,0 +1,77 @@
+import streamlit as st
+import pandas as pd
+import torch
+from torch import nn
+from torchvision import transforms, models
+from PIL import Image
+import altair as alt
+from collections import OrderedDict
+import gc
+
```

Comment:
I run into a Mac specific error - I have come across this error before and the only solution I have found - its not ideal but it means including - 
```
import os

os.environ["KMP_DUPLICATE_LIB_OK"] = "True"
```

Reply:
Thanks for the suggestion! This seems like an package conflict issue. Given that this error has not occurred during deployment, I will keep the code as is.

Reply:
ok! Just note that I am unable to run the app locally without the above.